### DISCUSSION TITLE
A vulnerability was found in ImageMagick. This security flaw causes a remote code execution vulnerability in OpenBlob with "--enable-pipes" configured.

### DISCUSSION THREAD

**Discussion by sadper102** [dlemstra/Magick.NET#1427] [Help] [answered]
How to fix it to pass checkmarx?
https://devhub.checkmarx.com/cve-details/CVE-2023-34152/

**Comment by dlemstra** [comment]
I have no clue why this was added. This was also reported in this issue: https://github.com/dlemstra/Magick.NET/issues/1423

**Reply by sadper102** [reply]
![image](https://github.com/dlemstra/Magick.NET/assets/1622914/d0017e6e-e5ee-4ac7-942e-79504d104535)

**Reply by dlemstra** [reply, marked as answer]
I think you will need to contact Checkmarx about this? Their tooling is reporting an incorrect issue. But maybe we can cheat if I publish a new version? Does it say anywhere what the maximum version is that they think includes this issue?

**Reply by dlemstra** [reply]
This should now be resolved.

**Reply by sadper102** [reply]
Still have been detected.
![Capture](https://github.com/dlemstra/Magick.NET/assets/1622914/d9bef5d1-8443-4153-b5ef-57d78ab87aaa)
